config/prometheus-collectors.php

```diff
@@ -15,6 +15,10 @@
         'default',
     ],
 
+    'queueLabels' => [
+        'default',
+    ],
```

Comment:
I'm thinking we should just merge this array and the one above, with either the keys or the values being the labels to use. What do you think?